src/app/features/search/components/search-form/search-form.component.html

```diff
@@ -0,0 +1,25 @@
+<form class="container" [formGroup]="searchForm" (ngSubmit)="search()">
+  <ng-container [ngTemplateOutlet]="input" [ngTemplateOutletContext]="{ title: 'from' }"></ng-container>
+  <ng-container [ngTemplateOutlet]="input" [ngTemplateOutletContext]="{ title: 'to' }"></ng-container>
+  <tui-input-date-time formControlName="date" timeMode="HH:MM"> Choose date and time </tui-input-date-time>
+  <button tuiButton appearance="primary" size="m" type="submit">Search</button>
+</form>
+
+<ng-template #input let-title="title">
+  <tui-input *tuiLet="cities() as cities" [formControl]="$any(searchForm.controls)[title]" class="search-input">
```

Comment:
can we use a standard `tui` classes for responsibility design ? like this?
![image](https://github.com/user-attachments/assets/bf999866-a45e-42e5-b8d0-52c35e9d3eb5)
![image](https://github.com/user-attachments/assets/0d5523ce-7fa4-4aaa-b158-a671679193ab)
![image](https://github.com/user-attachments/assets/30d250a7-2e26-4309-8778-d33838692c95)

Reply:
Thank you so much!